docs/pages/demo/datetime-picker/CustomDateTimePicker.example.jsx

```diff
@@ -2,6 +2,7 @@ import React, { useState } from 'react';
 import AlarmIcon from '@material-ui/icons/Alarm';
 import SnoozeIcon from '@material-ui/icons/Snooze';
 import ClockIcon from '@material-ui/icons/AccessTime';
+import PenIcon from '@material-ui/icons/Edit';
```

Comment:
For consistency with the two imports above?

```suggestion
import AccessIcon from '@material-ui/icons/AccessTime';
import EditIcon from '@material-ui/icons/Edit';
```